src/modules/auto_claim/index.js

```diff
@@ -81,19 +81,30 @@ class AutoClaimModule {
   async load() {
     const autoClaim = settings.get(SettingIds.AUTO_CLAIM);
     const shouldAutoClaim = hasFlag(autoClaim, AutoClaimFlags.DROPS);
-    const client = window.__twitch_pubsub_client;
-    const currentUser = getCurrentUser();
-    if (client == null || currentUser == null) {
-      return;
-    }
-    const topicId = `onsite-notifications.${currentUser.id}`;
-    if (!shouldAutoClaim && listenerAttached) {
-      listenerAttached = false;
-      client.topicListeners.removeListener(topicId, handleMessage);
```

Comment:
the old system was built around listening to the already-established topic listener.

it looks like Twitch is moving away from pubsub, thus why this is no longer being listened to here. we likely should move to the new hermes system instead.